druid-shell/src/platform/x11/keycodes.rs

```diff
@@ -1,3 +1,335 @@
 // TODO(x11/initial_pr): implement keycodes.rs, or re-TODO
 // TODO(x11/initial_pr): What type should RawKeyCode be? or re-TODO
-pub type RawKeyCode = i32;
\ No newline at end of file
+// pub type RawKeyCode = i32;
+
+// Copyright 2019 The xi-editor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// GTK keycode handling.
```

Comment:
I think this should be:

```suggestion
// X11 keycode handling.
```

Reply:
Done.